package.json

```diff
@@ -159,6 +159,27 @@
       "storage",
       "tabs"
     ],
+    "theme_icons": [{
```

Comment:
`theme_icons` is not a top-level property, but part of `browser_action`. Since it's a vector image (SVG), it suffices to specify just one size: 32.